src/CMakeLists.txt

```diff
@@ -135,17 +135,17 @@ set(Mapper_Common_SRCS
  
  compass.cpp
 
- file_format.cpp
- file_format_registry.cpp
- file_import_export.cpp
- file_format_native.cpp
- file_format_ocad8.cpp
- file_format_xml.cpp
- 
- fileformats/ocd_file_export.cpp
- fileformats/ocd_file_format.cpp
- fileformats/ocd_file_import.cpp
- fileformats/ocd_types.cpp
+ file_formats/file_format.cpp
+ file_formats/file_format_registry.cpp
+ file_formats/file_import_export.cpp
+ file_formats/file_format_native.cpp
+ file_formats/file_format_ocad8.cpp
+ file_formats/file_format_xml.cpp
+
+ file_formats/ocad/ocd_file_export.cpp
+ file_formats/ocad/ocd_file_format.cpp
+ file_formats/ocad/ocd_file_import.cpp
+ file_formats/ocad/ocd_types.cpp
```

Comment:
The file_format_* naming scheme of the other files is wrong. It is a poor substitute for using folders (which we do add now). In contrast, ocd_file_* were created by me with file names which match the class names, in a folder which is meant to take up all file formats, on the occassion of refactoring.

So now you know that you have to rename most files which you move to a folder.